packages/api-client/src/layouts/Web/ApiClientWeb.vue

```diff
@@ -0,0 +1,96 @@
+<script setup lang="ts">
+import { TheCommandPalette } from '@/components/CommandPalette'
+// TODO: Disabled until we polished the UI.
+// import { ImportCollectionListener } from '@/components/ImportCollection'
+import SideNav from '@/components/SideNav/SideNav.vue'
+import { useDarkModeState } from '@/hooks'
+import { handleHotKeyDown } from '@/libs'
+import { useWorkspace } from '@/store'
+import { addScalarClassesToHeadless } from '@scalar/components'
+import { getThemeStyles } from '@scalar/themes'
+import { ScalarToasts } from '@scalar/use-toasts'
+import {
+  computed,
+  onBeforeMount,
+  onBeforeUnmount,
+  onMounted,
+  watchEffect,
+} from 'vue'
+import { RouterView } from 'vue-router'
+
+onMounted(() => {
+  watchEffect(() => {
+    document.body.classList.toggle('dark-mode', isDark.value)
+    document.body.classList.toggle('light-mode', !isDark.value)
+  })
+})
+
+const { isDark } = useDarkModeState()
+const workspaceStore = useWorkspace()
+
+// Ensure we add our scalar wrapper class to the headless ui root
+onBeforeMount(async () => {
+  addScalarClassesToHeadless()
+})
```

Comment:
```suggestion
onBeforeMount(() => addScalarClassesToHeadless())
```
😄

Reply:
Yep that makes sense, tbh I copied this from the App layout

Reply:
barely makes a difference, just slightly cleaner looking.

Reply:
I do notice we use async in a few places where we don't really wait for anything 😄

Reply:
I feel like we should have a linter for that 🤔